metrics.proto

```diff
@@ -0,0 +1,109 @@
+syntax = "proto3";
+
+package io.containerd.cgroups.v1;
+
+import "gogoproto/gogo.proto";
+
+message Metrics {
+	map<string, HugetlbStat> hugetlb = 1;
```

Comment:
Does this map page size to the metrics? Should that just a field?

Reply:
Yes, page sizes

Reply:
I'm not sure it can be a field because they can change based on a system

Reply:
I was just thinking a string field `pagesize` instead of a map.

Reply:
we can do that